data/tokens.json

```diff
@@ -411,18 +323,151 @@
+            "fontFamily": "{typography.fontFamily.sansSerif}",
+            "fontWeight": "{typography.weight.heading}",
+            "lineHeight": "{typography.lineHeight.normal}",
+            "fontSize": "{typography.size.desktop.h4}",
+            "letterSpacing": "{typography.letterSpacing.base}",
+            "paragraphSpacing": "{typography.paragraphSpacing.base}",
+            "textDecoration": "{typography.textDecoration.base}",
+            "textCase": "{typography.textCase.base}"
+          },
+          "type": "typography"
+        },
+        "h5": {
+          "value": {
+            "fontFamily": "{typography.fontFamily.sansSerif}",
+            "fontWeight": "{typography.weight.heading}",
+            "lineHeight": "{typography.lineHeight.normal}",
+            "fontSize": "{typography.size.desktop.body}",
+            "letterSpacing": "{typography.letterSpacing.base}",
+            "paragraphSpacing": "{typography.paragraphSpacing.base}",
+            "textDecoration": "{typography.textDecoration.base}",
+            "textCase": "{typography.textCase.base}"
+          },
+          "type": "typography"
+        },
+        "body": {
+          "value": {
+            "fontFamily": "{typography.fontFamily.sansSerif}",
+            "fontWeight": "{typography.weight.base}",
+            "lineHeight": "{typography.lineHeight.normal}",
+            "fontSize": "{typography.size.desktop.body}",
+            "letterSpacing": "{typography.letterSpacing.base}",
+            "paragraphSpacing": "{typography.paragraphSpacing.base}",
+            "textDecoration": "{typography.textDecoration.base}",
+            "textCase": "{typography.textCase.base}"
+          },
+          "type": "typography"
+        },
+        "footnote": {
+          "value": {
+            "fontFamily": "{typography.fontFamily.sansSerif}",
+            "fontWeight": "{typography.weight.base}",
+            "lineHeight": "{typography.lineHeight.small}",
+            "fontSize": "{typography.size.desktop.footnote}",
+            "letterSpacing": "{typography.letterSpacing.base}",
+            "paragraphSpacing": "{typography.paragraphSpacing.base}",
+            "textDecoration": "{typography.textDecoration.base}",
+            "textCase": "{typography.textCase.base}"
+          },
+          "type": "typography"
+        },
+        "caption": {
+          "type": "typography",
+          "value": {
+            "fontFamily": "{typography.fontFamily.sansSerif}",
+            "fontWeight": "{typography.weight.base}",
+            "lineHeight": "{typography.lineHeight.small}",
+            "fontSize": "{typography.size.desktop.caption}",
+            "letterSpacing": "{typography.letterSpacing.base}",
+            "paragraphSpacing": "{typography.paragraphSpacing.base}",
+            "textDecoration": "{typography.textDecoration.base}",
+            "textCase": "{typography.textCase.base}"
+          }
+        }
+      },
+      "mobile": {
+        "h1": {
```

Comment:
I think this is a really good call, especially if we are finding we need to tweak the line heights of some of these on mobile 👍🏻